.eslintrc.json

```diff
@@ -67,6 +68,15 @@
       { "varsIgnorePattern": "^_", "argsIgnorePattern": "^_" }
     ],
     "@typescript-eslint/consistent-type-imports": "warn",
+    "lodash/import-scope": ["warn", "method"],
+    "lodash/preferred-alias": "warn",
+    "lodash/callback-binding": "warn",
+    "lodash/collection-method-value": "warn",
+    "lodash/collection-return": "warn",
+    "lodash/no-double-unwrap": "warn",
+    "lodash/no-extra-args": "warn",
+    "lodash/no-unbound-this": "warn",
+    "lodash/unwrap": "warn",
```

Comment:
This is quite a lot of customization. Is there no good configuration (`recommended`, `canonical`, `v3`, ...) we could use for `extends`?

Reply:
These are the [possible errors group from `eslint-plugin-lodash`](https://github.com/wix/eslint-plugin-lodash#possible-errors). Unfortunately, there's no extendable set without including several other customizations which I don't want, e.g. preferring lodash's functions over native methods.

Reply:
Hmm okay. Then it is what it is.